client/utils/format-ssr.js

```diff
@@ -0,0 +1,334 @@
+	for ( const [ tableName, tableDetail ] of Object.entries( tables ) ) {
+		result += tableName;
+		if ( tableDetail ) {
+			result += `: Data: ${ tableDetail.data }MB + Index: ${ tableDetail.index }MB + Engine ${ tableDetail.engine }`;
+		}
+		result += '\n';
+	}
+	return result;
+}
+
+function printPostTypeCounts( postTypeCounts ) {
+	let result = '';
+	if ( postTypeCounts ) {
+		result = '### Post Type Counts ###\n';
+		postTypeCounts.forEach( ( postType ) => {
+			result += `\n${ postType.type }: ${ postType.count }`;
+		} );
+	}
+	return result;
+}
+
+function printPlugins( plugins, header ) {
+	let result = '';
+	if ( header && 0 < plugins.length ) {
+		result = '\n### ' + header + ' (' + plugins.length + ')\n\n';
+	}
+	plugins.forEach( ( plugin ) => {
+		const currentVersion = plugin.version;
+		result += `${ plugin.name }: by ${ plugin.author_name } - ${ currentVersion }`;
+		const latestVersion = plugin.version_latest;
+		if (
+			currentVersion &&
+			latestVersion &&
+			currentVersion !== latestVersion
+		) {
+			result += ` (update to version ${ latestVersion } is available)`;
+		}
+		result += '\n';
+	} );
+	return result;
+}
+
+function printPages( pages ) {
+	let result = '';
+	pages.forEach( ( page ) => {
+		result += page.page_name + ': ';
+		let foundError = false;
+		if ( ! page.page_set ) {
+			result += CROSS_MARK + ' Page not set';
+			foundError = true;
+		} else if ( ! page.page_exists ) {
+			result +=
+				CROSS_MARK + ' Page ID is set, but the page does not exist';
+			foundError = true;
+		} else if ( ! page.page_visible ) {
+			result += CROSS_MARK + ' Page visibility should be public';
+			foundError = true;
+		} else if ( page.shortcode_required || page.block_required ) {
+			if ( ! page.shortcode_present && ! page.block_present ) {
+				result +=
+					CROSS_MARK +
+					` Page does not contain the ${ page.shortcode } shortcode or the ${ page.block } block`;
+				foundError = true;
+			}
+		}
+		if ( ! foundError ) {
+			result += 'Page ID #' + page.page_id;
+		}
+		result += '\n';
+	} );
+	return result;
+}
+
+function printTerms( arr ) {
+	let result = '';
+	Object.entries( arr ).forEach( ( [ key, value ] ) => {
+		result += value.toLowerCase() + ' (' + key + ')\n';
+	} );
+	return result;
+}
+
+/**
+ * Convert bytes into human-readable format. Resemble from PHP function size_format in WordPress core.
+ *
+ * @param {number} bytes Amount of bytes to be converted.
+ * @param {number} decimals Number of digits after the decimal. Default 0.
+ * @return {string} Human-readable string.
+ */
+function formatSize( bytes, decimals = 0 ) {
+	const KB_IN_BYTES = 1024;
+	const MB_IN_BYTES = KB_IN_BYTES * 1024;
+	const GB_IN_BYTES = MB_IN_BYTES * 1024;
+	const TB_IN_BYTES = GB_IN_BYTES * 1024;
+
+	const quant = [
+		[ 'TB', TB_IN_BYTES ],
+		[ 'GB', GB_IN_BYTES ],
+		[ 'MB', MB_IN_BYTES ],
+		[ 'KB', KB_IN_BYTES ],
+		[ 'B', 1 ],
+	];
+
+	if ( 0 === bytes ) {
+		return '0 B';
+	}
+
+	for ( const [ unit, mag ] of quant ) {
```

Comment:
Not sure what `mag` is the abbreviation of? `bytes_amount`?

Reply:
I think it's `magnitude of orders` for data https://en.wikipedia.org/wiki/Orders_of_magnitude_(data)

But yeah, I am changing it to `bytesPerUnit`